components/screens/TelemetryScreen.tsx

```diff
@@ -0,0 +1,50 @@
+import {createNativeStackNavigator, NativeStackScreenProps} from '@react-navigation/native-stack';
+import {TabNavigatorParamList, TelemetryStackParamList} from 'routes';
+import {StyleSheet, View} from 'react-native';
+import {TelemetryStationData} from 'components/TelemetryStationData';
+import React from 'react';
+import {TelemetryStationMap} from 'components/TelemetryStationMap';
+
+const TelemetryStack = createNativeStackNavigator<TelemetryStackParamList>();
+export const TelemetryTabScreen = ({route}: NativeStackScreenProps<TabNavigatorParamList, 'Weather Data'>) => {
+  const {center_id, date} = route.params;
+  return (
+    <TelemetryStack.Navigator initialRouteName="telemetryStations">
+      <TelemetryStack.Screen
+        name="telemetryStations"
+        component={TelemetryScreen}
+        initialParams={{center_id: center_id, date: date}}
+        options={() => ({title: `${center_id} Telemetry Stations`})}
```

Comment:
if i switch between centers and the center has no weather stations we can find, it defaults back to wa seattle centric map view, or sometimes will show wa stations randomly. not sure what going on with that - but maybe we should default to the zone of the center for this page no matter what? just random thoughts

Reply:
Yeah, I noticed that bad behavior - unrelated to this PR but we need to figure out how to gracefully handle that.